src/alire/os_linux/alire-platform.adb

```diff
@@ -30,12 +32,16 @@ package body Alire.Platform is
    begin
       if Distro_Cached then
          return Cached_Distro;
+      elsif not GNAT.OS_Lib.Is_Regular_File (OS_Identity_File) then
+         Distro_Cached := True;
+         Cached_Distro := Distro_Unknown;
+         return Cached_Distro;
       else
          declare
             use Utils;
             Release : constant Utils.String_Vector :=
                         Subprocess.Checked_Spawn_And_Capture
-                ("cat", Empty_Vector & "/etc/os-release");
+                ("cat", Empty_Vector & OS_Identity_File);
```

Comment:
We could read the file directly instead of spawning `cat`.

Reply:
Yes, this is really the result of a low-glucose moment :sweat_smile: